docs/docs/hardware.md

```diff
@@ -21,9 +21,9 @@ That being said, there are currently only a few specific [boards](/docs/faq#what
 - [QMK Proton-C](https://qmk.fm/proton-c/) (`proton_c`)
 
 ## Keyboard Shields
-
```

Comment:
@KingCoinless I think your build is failing because you removed this line. Could you add it back?

Reply:
Thank you for waiting on my update. I replaced the blank line, and am going to make another commit shortly.